recipe/meta.yaml

```diff
@@ -1,47 +1,57 @@
 {% set name = "python-magic" %}
 {% set version = "0.4.27" %}
-{% set hash_val = "c1ba14b08e4a5f5c31a302b7721239695b2f0f058d125bd5ce1ee36b9d9d3c3b" %}
+{% set hash_val = "3978a25d43d9a7b8a89ae9d726bd4962fc90dc4f69ae852e399f3c56d4b0bd63" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ahupp/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
   sha256: {{ hash_val }}
-  patches:
-    # - discover-libmagic-dylib.patch  # [osx]
-      - 0001-load-magic-file.patch
+
 
 build:
-  number: 3
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [win]
+  number: 4
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<38 or win]
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
     - libmagic
```

Comment:
Is `python-magic` trying to load the system `libmagic` not from conda on `osx` platforms? See https://github.com/ahupp/python-magic/blob/0.4.27/magic/loader.py#L13-L17

conda build will look at `libmagic` files on `osx-64` inside `$PREFIX/` here:
```
include/magic.h
lib/libmagic.1.dylib
lib/libmagic.dylib
lib/pkgconfig/libmagic.pc
share/misc/magic.mgc
```

See https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain%2Fosx-64%2Flibmagic-5.39-he369b6f_1.conda

Reply:
@skupr-anaconda do you mean that we should patch the paths into using `$PREFIX/`?

CC @psteyer

Reply:
I think the removed patches should be reverted back as they needed for picking up correct conda libraries instead of finding the system libs

Reply:
I added a patch that I think should fix the absolute path in the source code for osx.

Thanks @lorepirri!